### DISCUSSION TITLE
Radar Chart invert axis

### DISCUSSION THREAD

**Discussion by vaibhav-pan** [imaNNeo/fl_chart#1355] [Q&A]
Is there a way to invert the radial axis on Radar chart? Like instead of going from 0 to 10, center to outside, it goes 10 to 0?

**Comment by imaNNeo** [comment]
It is not a built-in functionality.
But you can do that manually

**Comment by vaibhav-pan** [comment]
How can that be done manually

**Reply by vaibhav-pan** [reply]
@imaNNeo I would appreciate any pointer

**Reply by imaNNeo** [reply]
by reversing your data

**Reply by vaibhav-pan** [reply]
If you mean mirroring the data to negative x axis, it still would go from -10 at the center to 0 at the outermost circle. Is there a way to not show the negative sign. 
Unless you meant something different

**Reply by imaNNeo** [reply]
You can use `getTitle` in [RadarChartData](https://github.com/imaNNeo/fl_chart/blob/master/repo_files/documentations/radar_chart.md#radarchartdata) to provide customized titles.

**Reply by vaibhav-pan** [reply]
But i require the tick values shown as well, not just the title outside
